raiden/storage/sqlite.py

```diff
@@ -1617,3 +1639,11 @@ def flag_light_client_as_pending_for_deletion(self, address):
             last_id = cursor.lastrowid
 
         return last_id
+
+    def update(self, update_sql: str, params: Any) -> int:
+        """ Aux method to generalize the update calls """
+        with self.write_lock, self.conn:
+            cursor = self.conn.cursor()
+            cursor.execute(update_sql, params)
+            last_id = cursor.lastrowid
+        return last_id
```

Comment:
why is this called `update`? couldn't it just execute _any_ SQL it receives?

Reply:
actually it just executes an update, since it gets the lastrowid, we could improve this to be more generic style to work not just with updates, maybe return the cursor, i don't know but at least this is better than copy/pasting the segment of code to query the database, at least the method call is much more cleaner.